ani.ter-markosyan/repo/Lesson_3/exercise_08/exercise_08.py

```diff
@@ -0,0 +1,6 @@
+first_number = int(input("Enter a number: "))
+second_number = int(input("Enter a number: "))
+third_number = int(input("Enter a number: "))
+total = (first_number + second_number + third_number)
+average = ((first_number + second_number + third_number) / 3)
+print(total,round(average,2))
```

Comment:
CSW: User message should describe which variables equal to the corresponded value